products/memgraph/upcoming/tutorials/marvel-universe.md

```diff
@@ -1,5 +1,12 @@
 ## Marvel Comic Universe Social Network
 
+This article is a part of a series intended to show how to use Memgraph
+on real-world data to retrieve some interesting and useful
+information.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials-overview.md).
+
```

Comment:
Please remove the list below, sorry for not detecting this in the first pass...

Reply:
It's not like we're waiting for 2-hour build on each push :smile: